deployment-templates/__init__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2017 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Forseti Deployment Templates."""
+
+try:
```

Comment:
How do we decide when/where to put this try/catch?

Reply:
Only on namespaces we've declared in setup.py:

https://github.com/GoogleCloudPlatform/forseti-security/blob/2.0-dev/setup.py#L31

Reply:
It seems like we have this try/catch in deployment-templates/__init__.py and tests/__init__.py but these 2 folders are not declared in the NAMESPACE_PACKAGES?

Reply:
Apologies, yes we still need to declare namespace when we are in a different root directory: `/google/` versus `/test/`

Reply:
Thanks for the clarifications! Everything else looks good.